docs/content/docs/tutorials/sagemaker-example.md

```diff
@@ -0,0 +1,346 @@
+
+```bash
+export OIDC_ROLE_NAME=ack-controller-role-$CLUSTER_NAME
+aws --region $AWS_DEFAULT_REGION iam create-role --role-name $OIDC_ROLE_NAME --assume-role-policy-document file://trust.json
+```
+
+Attach the AmazonSageMakerFullAccess Policy to the IAM Role to ensure that your SageMaker service controller has access to the appropriate resources. 
+
+```bash
+aws --region $AWS_DEFAULT_REGION iam attach-role-policy --role-name $OIDC_ROLE_NAME --policy-arn arn:aws:iam::aws:policy/AmazonSageMakerFullAccess
+export IAM_ROLE_ARN_FOR_IRSA=$(aws --region $AWS_DEFAULT_REGION iam get-role --role-name $OIDC_ROLE_NAME --output text --query 'Role.Arn')
+echo $IAM_ROLE_ARN_FOR_IRSA
+```
+
+For more information on authorization and access for ACK service controllers, including details regarding recommended IAM policies, see [Configure Permissions][configure-permissions].
+
+### Install the SageMaker ACK service controller
+
+Get the SageMaker Helm chart and make it available on the client machine with the following commands:
+
+```bash
+export HELM_EXPERIMENTAL_OCI=1
+export SERVICE=sagemaker
+export LATEST_RELEASE_VERSION=`curl -sL https://api.github.com/repos/aws-controllers-k8s/sagemaker-controller/releases/latest | grep '"tag_name":' | cut -d'"' -f4`
+export RELEASE_VERSION=${LATEST_RELEASE_VERSION:-v0.0.4}
+export CHART_EXPORT_PATH=/tmp/chart
+export CHART_REPO=public.ecr.aws/aws-controllers-k8s/$SERVICE-chart
+export CHART_REF=$CHART_REPO:$RELEASE_VERSION
+export ACK_K8S_NAMESPACE=ack-system
+
+mkdir -p $CHART_EXPORT_PATH
+helm chart pull $CHART_REF
+helm chart list
+helm chart export $CHART_REF --destination $CHART_EXPORT_PATH
+```
+
+Update the Helm chart values for a cluster-scoped installation. 
+
+```bash
+# Update the following values in the Helm chart
+cd $CHART_EXPORT_PATH/$SERVICE-chart
+yq e '.aws.region = env(AWS_DEFAULT_REGION)' -i values.yaml
+yq e '.aws.account_id = env(AWS_ACCOUNT_ID)' -i values.yaml
+yq e '.serviceAccount.annotations."eks.amazonaws.com/role-arn" = env(IAM_ROLE_ARN_FOR_IRSA)' -i values.yaml
+cd -
+```
+
+Install the SageMaker ACK service controller and relevant custom resource definitions (CRDs) with the Helm chart. 
+```bash
+helm install -n $ACK_K8S_NAMESPACE --create-namespace ack-$SERVICE-controller \
+ $CHART_EXPORT_PATH/$SERVICE-chart
+```
+
+Verify that the CRDs and Helm charts were deployed with the following commands:
+```bash
+kubectl get crds
+kubectl get pods -n $ACK_K8S_NAMESPACE
+```
+
+## Train an XGBoost model
+
+### Prepare your data
+
+For training a model with SageMaker, we will need an S3 bucket to store the dataset and model training artifacts. For this example, we will use [MNIST](http://yann.lecun.com/exdb/mnist/) data stored in [LIBSVM](https://www.csie.ntu.edu.tw/~cjlin/libsvm/) format.
+
+First, create a variable for the S3 bucket:
+
+```bash
+export ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
+# Specify your service region
+export SERVICE_REGION=us-east-1
```

Comment:
the default region for the controller is defined here - https://github.com/aws-controllers-k8s/community/pull/945/files#diff-785fe064cc3ba92455af3ccd1dd44ddd6714e1b9cc6c2a11b3bbdd12fdae4cd5R131.

If the region for the job/resource is different from the default controller region, we will need this - https://github.com/aws-controllers-k8s/sagemaker-controller#50-cross-region-resource-management

For this tutorial, recommend keeping everything in one region or change [the substituted region here](https://github.com/aws-controllers-k8s/community/pull/945/files#diff-785fe064cc3ba92455af3ccd1dd44ddd6714e1b9cc6c2a11b3bbdd12fdae4cd5R131) to `SERVICE_REGION`